common/models/Article.php

```diff
@@ -4,10 +4,11 @@
 
 use common\models\query\ArticleQuery;
 use trntv\filekit\behaviors\UploadBehavior;
-use Yii;
+use yii;
```

Comment:
Ah, it could be because of a bug in PHPStorm saying `Namespace reference is not case-sensitive` thinking `use Yii` is actually trying to use the `yii` namespace. Because it's case insensitive it works both ways.